bindings/flow/tester/Tester.actor.cpp

```diff
@@ -112,6 +112,8 @@ std::string tupleToString(Tuple const& tuple) {
 			str += format("%016llx%016llx", *(uint64_t*)u.getData().begin(), *(uint64_t*)(u.getData().begin() + 8));
 		} else if (type == Tuple::NESTED) {
 			str += tupleToString(tuple.getNested(i));
+		} else if (type == Tuple::VERSIONSTAMP) {
+			str += "\'" + tuple.getString(i).printable() + "\'";
```

Comment:
It would probably be good to have the versionstamp print as something more distinctive than just bytes.